source/server/admin/prometheus_stats.cc

```diff
@@ -236,6 +251,13 @@ uint64_t PrometheusStatsFormatter::statsAsPrometheus(
       outputStatType<Stats::Gauge>(response, used_only, regex, gauges,
                                    generateNumericOutput<Stats::Gauge>, "gauge", custom_namespaces);
 
+  if (export_text_readouts) {
```

Comment:
Instead of 2 params here, please make `text_readouts` an optional, or pass an empty one, when this behavior isn't desired, and remove the bool param.

Reply:
Done